### PR TITLE
Display content items on sub-topic pages

### PR DESCRIPTION
**Pull request by vanitabarrett**
Trello card: https://trello.com/c/TyogtYZZ/41-list-content-items-on-sub-topic-pages-prototype

[Example Page for Education Taxonomy](https://govuk-topic-pages-prototy-pr-9.herokuapp.com/topics/education)

screen shot 2018-01-31 at 11 17 37

### CODE REVIEW

File: spec/controllers/topics_controller_spec.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 require 'rails_helper'
 
 RSpec.describe TopicsController, type: :controller do
+  include ContentSchemaHelpers
```
Comment (leenagupte) [inline spec/controllers/topics_controller_spec.rb:4]: Do you need this here?

Comment (vanitabarrett) [reply spec/controllers/topics_controller_spec.rb:4]: @leenagupte I think so, it provides the `example_content_schema` method which we use in these tests: https://github.com/alphagov/govuk-topic-pages-prototype/pull/9/files#diff-4bcc73cef58214df200f492de3d9606fR23

Comment (leenagupte) [reply spec/controllers/topics_controller_spec.rb:4]: Oh! That's a later commit. The line is added in 3333032f955e85909e5a631f4c701c7d0e455dc3. I'd move this statement to the later commit.

File: app/models/rummager_search.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+class RummagerSearch
+  PAGE_SIZE_TO_GET_EVERYTHING = 1000
+
+  def initialize(content_id)
+    @content_id = content_id
```
Comment (leenagupte) [inline app/models/rummager_search.rb]: You can set an `attr_reader` for `content_id` which is the equivalent of wrapping `@content_id` in a getter method.  In `search_params` you could then call `content_id` without the `@`.

File: app/views/topics/show.html.erb [hunk 3]
```diff
@@ -1,13 +1,27 @@
 <%= render "govuk_component/title", title: @taxon.title %>
 
 <div>
-  <ul>
-    <% if @taxon.children? %>
+  <% if @taxon.children? %>
+    <h1>Sub Topics</h1>
```
Comment (leenagupte) [inline app/views/topics/show.html.erb]: Minor thing, but I think there should only be one H1 tag on a page.

File: app/views/topics/show.html.erb [hunk 4]
```diff
@@ -1,13 +1,27 @@
...
         </li>
       <% end %>
-    <% end %>
-  </ul>
+    </ul>
+  <% end %>
+</div>
+
+<div>
+  <% if @tagged_content.length > 0 %>
```
Comment (leenagupte) [inline app/views/topics/show.html.erb:17]: I think you could squash up this update to the view, listing the tagged content, up to commit 5f86d7d8a4e939516041c91ef6b7aa16432c102b. If you do that `@tagged_content` will be defined and used in the same commit, and the story of the change will flow better.

I'm happy for the CSS changes and the update to the startup script to remain in separate commits.

**Review by leenagupte — CHANGES_REQUESTED** [review]
The change as a whole looks good. Just a couple of minor comments, mostly related to commit structure.

**Review by leenagupte — APPROVED** [review]
👍  Looks good.

Merge away!

**Pull request merged**